mmv1/third_party/terraform/services/dialogflowcx/resource_dialogflow_cx_test_case_test.go

```diff
@@ -0,0 +1,255 @@
+  parent       = google_dialogflow_cx_agent.agent.id
+  display_name = "MyIntent"
+  priority     = 1
+  training_phrases {
+    parts {
+      text = "training phrase"
+    }
+    repeat_count = 1
+  }
+}
+
+resource "google_dialogflow_cx_test_case" "basic_test_case" {
+  parent       = google_dialogflow_cx_agent.agent.id
+  display_name = "MyTestCase"
+
+  test_config {
+    tracking_parameters = []
+    flow                = google_dialogflow_cx_agent.agent.start_flow
+  }
+
+  test_case_conversation_turns {
+    user_input {
+      input {
+        language_code = "en"
+        text {
+          text = "training phrase"
+        }
+      }
+    }
+    virtual_agent_output {
+      text_responses {
+        text = ["Training phrase response"]
+      }
+    }
+  }
+}`, context)
+}
+
+func testAccDialogflowCXTestCase_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_dialogflow_cx_agent" "agent" {
+  display_name               = "tf-test-dialogflowcx-agent%{random_suffix}"
+  location                   = "global"
+  default_language_code      = "en"
+  supported_language_codes   = ["fr", "de", "es"]
+  time_zone                  = "America/New_York"
+  description                = "Example description."
+  avatar_uri                 = "https://storage.cloud.google.com/dialogflow-test-host-image/cloud-logo.png"
+  enable_stackdriver_logging = true
+  enable_spell_correction    = true
+  speech_to_text_settings {
+    enable_speech_adaptation = true
+  }
+}
+
+resource "google_dialogflow_cx_page" "page" {
+  parent       = google_dialogflow_cx_agent.agent.start_flow
+  display_name = "MyPage"
+
+  transition_routes {
+    intent = google_dialogflow_cx_intent.intent.id
+    trigger_fulfillment {
+      messages {
+        text {
+          text = ["Training phrase response"]
+        }
+      }
+    }
+  }
+
+  event_handlers {
+    event = "some-event"
+    trigger_fulfillment {
+      messages {
+        text {
+          text = ["Handling some event"]
+        }
+      }
+    }
+  }
+}
+
+resource "google_dialogflow_cx_intent" "intent" {
+  parent       = google_dialogflow_cx_agent.agent.id
+  display_name = "MyIntent"
+  priority     = 1
+  training_phrases {
+    parts {
+      text = "training phrase"
+    }
+    repeat_count = 1
+  }
+}
+
+resource "google_dialogflow_cx_test_case" "basic_test_case" {
+  parent       = google_dialogflow_cx_agent.agent.id
+  display_name = "MyTestCase"
+  tags         = ["#tag1"]
+  notes        = "demonstrates a simple training phrase response"
+
+  test_config {
+    tracking_parameters = ["some_param"]
+    page                = google_dialogflow_cx_page.page.id
+  }
+
+  test_case_conversation_turns {
+    user_input {
+      input {
+        language_code = "en"
+        text {
+          text = "training phrase"
+        }
+      }
+      injected_parameters       = jsonencode({ some_param = "1" })
+      is_webhook_enabled        = false
```

Comment:
`false` and empty are generally considered the same by Terraform - would it be possible to set these to `true` instead?

Reply:
Oops, good point, done.